lib/squad_goals/app.rb

```diff
@@ -20,20 +20,61 @@ class App < Sinatra::Base
       use Rack::SslEnforcer
     end
 
+    configure :development do
+      Dotenv.load
+      Octokit.middleware = Faraday::RackBuilder.new do |builder|
+        builder.response :logger
+        builder.use Octokit::Response::RaiseError
+        builder.adapter Faraday.default_adapter
+      end
+    end
+
+    configure do
+      set :organization, Organization.new(ENV['GITHUB_ORG_ID'])
+      set :client, Octokit::Client.new(access_token: ENV['GITHUB_TOKEN'])
+      set :dalli, Dalli::Client.new((ENV['MEMCACHIER_SERVERS'] || 'localhost:11211').split(','),
+                                    username: ENV['MEMCACHIER_USERNAME'],
+                                    password: ENV['MEMCACHIER_PASSWORD'],
+                                    failover: true,
+                                    socket_timeout: 1.5,
+                                    socket_failure_delay: 0.2,
+                                    exires_in: MEMCACHE_TTL)
```

Comment:
The home page now requires one call to get the teams list, and one call per team to get team members, all unauthenticated. With Jekyll, we're talking 6 teams, so 7 calls per page load, which I feared we might run into API rate limits. Since we're running on Heroku, I didn't know a better way to cache between requests (and choice Memcache over Redis for simplicity of being able to use Marshal over JSON).